internal/commands/pull/inputs.go

```diff
@@ -14,6 +14,13 @@ import (
 	"github.com/mitchellh/go-homedir"
 )
 
+const (
+	flagIncludePackageJSON  = "include-package-json"
+	flagIncludeDependencies = "include-dependencies"
+
+	errDependencyFlagConflictTemplate = `cannot use both "%s" and "%s" at the same time`
```

Comment:
[style] can we remain consistent with the grouping of these consts?  we have both separate const blocks and separated within the same const block for this stuff, and then some are defined in the command file and some in the inputs file.

just to throw an idea out there, let's keep all "flag" consts in the command files.  the "error template" consts can live in separate const blocks, in the file they're used (for push/pull, the inputs file.  for app diff, the command file)